### ISSUE TITLE
Report

### ISSUE THREAD

**Issue by jkarolczak** [jkarolczak/image-inpainting#2]
- [x] description of the data set, with a few image examples
- [x] description of the problem
- [x] description of used architectures with graphs; model analysis: size in memory, number of parameters,   - @jkarolczak 
- [x] description of the training - @konradszewczyk 
- [x] description of used metrics, loss, and evaluation - @konradszewczyk 
- [x] plots: training and validation loss, metrics - @jkarolczak 
- [x] used hyperparameters along with an explanation of each why such value was chosen - @jkarolczak 
- [x] comparison of models - @konradszewczyk @jkarolczak 
- [x] list of libraries and tools used
- [x] a description of the runtime environment
- [x] training and inference time, - @jkarolczak 
- [x] preparation of a bibliography - the bibliography should contain references to the data set (preferably the article in which the collection was presented) and all scientific works and studies, including websites with tips on the solution.
analyzes obtained from additional tools

**Comment by jkarolczak** [comment]
- [x] - yaml parameters - @konradszewczyk
